### PR TITLE
[fix bug 1380845] Archive Persona privacy policy

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Moves and redirects Persona privacy policy pages to the privacy archive.
- Removes Person app from bedrock.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1380845

## Testing
Demo:
- https://www-demo1.allizom.org/en-US/privacy/archive/
- https://www-demo1.allizom.org/en-US/privacy/archive/persona/2017-07/

Redirects are also in place for the old URLs:
- https://www-demo1.allizom.org/en-US/persona/privacy-policy/
- https://www-demo1.allizom.org/en-US/persona/terms-of-service/

Check I didn't miss any URL's or links to Persona?

### CODE REVIEW

**Comment by alexgibson** [comment]
[Confirmed in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1380845#c10) that this looks ok for legal. This should be good for review.

File: bedrock/privacy/urls.py [hunk 1]
```diff
@@ -38,4 +38,6 @@
     page('/archive/hello/2016-03', 'privacy/archive/hello-2016-03.html'),
     page('/archive/thunderbird/2010-06', 'privacy/archive/thunderbird-2010-06.html'),
     page('/archive/websites/2013-08', 'privacy/archive/websites-2013-08.html'),
+    page('/archive/websites/2013-08', 'privacy/archive/websites-2013-08.html'),
```
Comment (pmac) [inline bedrock/privacy/urls.py]: From the diff here it looks like you've duplicated this line by accident.

**Comment by pmac** [comment]
Merge at will after fixing that line.

File: bedrock/privacy/urls.py (hunk 1 above)
Comment (alexgibson) [inline bedrock/privacy/urls.py]: good spot 👍

**Pull request merged**